src/System.Runtime.Caching/src/System.Runtime.Caching.csproj

```diff
@@ -15,7 +15,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp2.0-Windows_NT-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
-  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
+  <ItemGroup>
```

Comment:
You don't need to generate a turd assembly anymore. You can remove `<GeneratePlatformNotSupportedAssemblyMessage>` in the .csproj.